mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/ExceptionEventRequest.cs

```diff
@@ -3,10 +3,10 @@
 
 namespace Mono.Debugger.Soft
 {
-	public sealed class ExceptionEventRequest : EventRequest {
+	public class ExceptionEventRequest : EventRequest {
```

Comment:
Wouldn't be easier to add the new property to the existing class ? Here and everywhere else ? And just pass it when the debuggee has a newer protocol version ?

Reply:
The thing is, VSMac will continue using the older exception filter, because VsMac features are supported with that, but VSWin needs another implementation, because of that I decided to keep both. But I could keep both and receive a flag to decide which one should I use... I don't know if this would be easier to understand.

Reply:
The flag would have the default value which is the old behavior and the ide could set it to a different value to get the new behavior.

Reply:
Done